AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,7 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+neilpatel.com###bottom-box
```

Comment:
@wout-codes please block cookie popup too

Reply:
Reproduced and fixed.

Reply:
and please remove the old rule for cookie =)